### PR TITLE
Add Scalabel format typed io

### PR DESCRIPTION
**Pull request by fyu**

### CODE REVIEW

File: bdd100k/eval/mot.py [hunk 1]
```diff
@@ -32,24 +41,30 @@
 IGNORE_CLASSES = ["trailer", "other person", "other vehicle"]
 
 
-def parse_objects(objects: List[DictAny]) -> List[np.ndarray]:
+def parse_objects(objects: List[Label]) -> List[np.ndarray]:
     """Parse objects under Scalable formats."""
     bboxes, labels, ids, ignore_bboxes = [], [], [], []
     for obj in objects:
+        box2d = obj.box2d
```
Comment (fyu) [inline bdd100k/eval/mot.py:48]: in `Label`, the underscore in `box_2d` is necessary. You can check the unit tests in scalabel.

File: bdd100k/common/utils.py [hunk 2]
```diff
@@ -84,7 +87,7 @@ def list_files(inputs: str) -> List[List[str]]:
     return files_list
 
 
-def read(inputs: str) -> List[List[DictAny]]:
+def read(inputs: str) -> List[List[Frame]]:
     """Read annotations from file/files."""
```
Comment (fyu) [inline bdd100k/common/utils.py:91]: We can use `scalabel.label.io` here

Comment (XiaLiPKU) [reply bdd100k/common/utils.py:91]: The JSON files we have in hand for BDD don't strictly follow the format definitions. But you can see some specific settings in this function

**Review by fyu — COMMENTED** [review]
giphy

**Pull request merged**